config.example.jsonc

```diff
@@ -0,0 +1,124 @@
+// DEV-SETUP: Make a copy of this file, and call it 'config.json'. Then remove all the comments below while following what they say to do!
+// - OPTIONAL settings can be left with their current values and the majority of the bot will work just fine
+// - REQUIRED settings must be set to a valid value for large parts of the bot to work, or start up in the first place.
+// All configs that already have values are designed to work with the container layout that results from using vscode dev containers. If youre developing by a different means then they will likely have to be different.
+{
+    "version": "4.5.1",
+    "cassandra": {
+        "contactPoints": [
+            "cassandra"
+        ],
+        "keyspace": "blargbot",
+        "username": "blargbot",
+        "password": "BlargPass123"
+    },
+    "rethink": {
+        "host": "rethinkdb",
+        "user": "blargbot",
+        "password": "BlargPass123",
+        "db": "blargbot",
+        "port": 28015
+    },
+    "postgres": {
+        "user": "postgres",
+        "pass": "password",
+        "database": "blargbot",
+        "sequelize": {
+            "host": "postgres",
+            "pool": {
+                "max": 15,
+                "min": 3,
+                "acquire": 60000,
+                "idle": 10000
+            }
+        }
+    },
```

Comment:
Variables like ports, passwords, db names, user names, etc. As they should match what is in the docker-compose, wouldn't it be better to have them in a single file (.env ?) instead of having to duplicate them in the yml and json files?

Reply:
Maybe if this was also being used for prod, but this is just being used for one specific way of development, and theres no real reason to ever change them in development. I figured this was lowest impact, especially seeing as I dont have access to update the config.json in the deployed environment to make this change